ProjectTodo/src/services/AuthService.js

```diff
@@ -0,0 +1,36 @@
+import BaseService from './BaseService';
+import { AsyncStorage } from 'react-native';
+
+const ENDPOINTS = {
+  LOGIN: '/login'
+};
+
+class AuthService extends BaseService {
+  constructor(props) {
+    super(props);
+  }
+
+  getUserFromToken(token) {
+    var base64Url = token.split('.')[1];
+    var base64 = base64Url.replace('-', '+').replace('_', '/');
+    return JSON.parse(window.atob(base64))['user'];
+  }
+
+  login = async data => {
+    try {
+      let response = await this.apiClient().post(ENDPOINTS.LOGIN, data);
+      var token = response['data']['access_token'];
+      var user = this.getUserFromToken(token);
+      this.apiClient().defaults.headers.common['Authorization'] =
```

Comment:
za ovo imas funkciju u httpService koja to radi za tebe pricao sam o tome tebi i Eleni